.github/workflows/python-build.yml

```diff
@@ -0,0 +1,31 @@
+name: Python Test
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v3
+    - name: Install dependencies
+      run: |
+        sudo apt-get install python3-dev
+        python -m pip install --upgrade pip
+        pip install pylint black
+        pip install -r requirements.txt
+    - name: Analysing the code with pylint
+      run: |
+        pylint $(git ls-files '*.py') --exit-zero 
+    - name: Run python black code formatter
+      uses: DataDog/action-py-black-formatter@v2.5
+      with:
+        fail_on_error: 'false'
+    - name: Build
+      run: |
+        python setup.py install
```

Comment:
If this command fails then workflow will let you know before merging the PR or on new push

Reply:
You can try running the workflow for this PR. It might help you understand it.